src/snowflake/snowpark/modin/plugin/extensions/series_extensions.py

```diff
@@ -55,7 +55,7 @@ def to_snowflake(
 
     See Also:
         - :func:`to_snowflake <snowflake.snowpark.modin.pandas.io.to_snowflake>`
-        - :func:`DataFrame.to_snowflake <snowflake.snowpark.modin.pandas.DataFrame.to_snowflake>`
```

Comment:
@sfc-gh-joshi i want to double check here those are just for doc link, it won't show up to user with the full path, right?

Reply:
That's correct. This determines the path of the method to link to.

I'll see about fixing the link to `to_snowflake` after this PR, it might have been broken for a while now.

![Screenshot 2024-09-06 at 12 06 37](https://github.com/user-attachments/assets/e35a1fbe-3f3e-4ecc-9abb-5e135ba686c8)